docs/UserGuide.md

```diff
@@ -411,21 +426,33 @@ Format: `undo` or `redo`
 
 Example:
 
-Let the initial state of the list contain these 3 students: Alex, Bernice, Charlotte.
+Let the initial state of the list contain these 3 students: Alex, Bernice, Charlotte.<br>
+
 ![Initial State](images/undocommandUG/initial_state.png)
 
-After `delete 1`, we will delete Alex and the list will not have Alex anymore.
+After `delete 1`, we will delete Alex and the list will not have Alex anymore.<br>
+
 ![Before Undo](images/undocommandUG/before_undo.png)
 
-After `undo`, the list will return to having Alex in it
+After `undo`, the list will return to having Alex in it<br>
+
 ![Initial State](images/undocommandUG/initial_state.png)
 
-After a `redo`, the list will return to the state where Alex was deleted
+After a `redo`, the list will return to the state where Alex was deleted<br>
+
 ![Before Undo](images/undocommandUG/before_undo.png)
 
 ### Mail commands
 [<sub><sup>Back to top</sup></sub>](#table-of-contents)
 
+<div markdown="span" class="alert alert-primary">:bulb: **Before you use mail commands :**
+Every system has a default mail application which opens up when `mailto` commands are invoked. If not, the
+operating system would request for the suitable application to respond with. If you wish to send mails through your
+preferred mail client, you may set it as the default application in the OS settings.
+</div>
```

Comment:
I think it might be a good idea to include this dependency in the quick start too, so the users know what to expect before they start using the application

Reply:
Okay!